docs/preview features/speech.md

```diff
@@ -6,7 +6,10 @@ Pre-requisites: Before you can connect a bot to Speech, be sure you've completed
 
 * You've built a basic bot in Composer
 * You've created a publishing profile and published your bot to Azure
-* You've [created a Cognitive Services key](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesAllInOne)
+* You've [created a Speech Service](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesSpeechServices) in the same DC Region of your Bot Registration (Reccomend to add to the same Resource Group if possible)
 * You've enabled the Speech connection in your bot's settings
-
-To verify your connection to the Speech service, [follow the instructions here](https://docs.microsoft.com/azure/bot-service/bot-service-channel-connect-directlinespeech?view=azure-bot-service-4.0#example).
+* Download the Windows [Voice Assistant Client](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/releases) (Only Available for Windows)
+* In Windows Voice Assistant Client put your Speech Key in Subscritpion Key under General Settings and your DC region in the Subscription key region. Then Save and Apply Profile.
```

Comment:
Subscritpion misspelled.
Check capitalization.
Then Save and Apply Profile. <- sentence fragment.

Reply:
i fixed this.